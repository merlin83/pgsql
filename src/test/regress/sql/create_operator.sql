--
-- OPERATOR DEFINITIONS
--
CREATE OPERATOR ## ( 
   leftarg = path,
   rightarg = path,
   procedure = path_inter,
   commutator = ## 
);

CREATE OPERATOR <% (
   leftarg = point,
   rightarg = circle,
   procedure = pt_in_circle,
   commutator = >=% 
);

CREATE OPERATOR @#@ (
   rightarg = int4,		-- left unary 
   procedure = int4fac 
);

CREATE OPERATOR #@# (
   leftarg = int4,		-- right unary
   procedure = int4fac 
);

CREATE OPERATOR #%# ( 
   leftarg = int4,		-- right unary 
   procedure = int4fac 
);

