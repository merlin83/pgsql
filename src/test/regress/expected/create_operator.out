QUERY: CREATE OPERATOR ## (
   leftarg = path,
   rightarg = path,
   procedure = path_inter,
   commutator = ##
);
QUERY: CREATE OPERATOR <% (
   leftarg = point,
   rightarg = widget,
   procedure = pt_in_widget,
   commutator = >% ,
   negator = >=%
);
QUERY: CREATE OPERATOR @#@ (
   rightarg = int4,		
   procedure = int4fac
);
QUERY: CREATE OPERATOR #@# (
   leftarg = int4,		
   procedure = int4fac
);
QUERY: CREATE OPERATOR #%# (
   leftarg = int4,		
   procedure = int4fac
);
